projects/demo/src/app/app.module.ts

```diff
@@ -26,7 +26,7 @@ import { AppComponent } from './app.component';
     MatButtonModule,
     MatCardModule,
     MatCheckboxModule,
-    MatCarouselModule,
+    MatCarouselModule.forRoot(),
```

Comment:
If `forRoot` is the way to import this module now, maybe it should be a nice idea to explain that in README.